cairo install fail

Cairo installation fails with the message

`cairo-xlib-surface.c:71:33: fatal error: X11/extensions/XShm.h: No such file or directory
 #include <X11/extensions/XShm.h>`

I'm not sure which dependency should be added.

--- a/packages/cairo.rb
+++ b/packages/cairo.rb
@@ -3,7 +3,7 @@ require 'package'
 class Cairo < Package
   description 'Cairo is a 2D graphics library with support for multiple output devices.'
   homepage 'https://www.cairographics.org'
-  version '1.14.10'
+  version '1.14.10-1'
   source_url 'https://www.cairographics.org/releases/cairo-1.14.10.tar.xz'
   source_sha256 '7e87878658f2c9951a14fc64114d4958c0e65ac47530b8ac3078b2ce41b66a09'
 

--- a/packages/cairo.rb
+++ b/packages/cairo.rb
@@ -2,7 +2,7 @@ require 'package'
 
 class Cairo < Package
   description 'Cairo is a 2D graphics library with support for multiple output devices.'
-  homepage 'https://www.cairographics.org/'
+  homepage 'https://www.cairographics.org'
   version '1.14.10'
   source_url 'https://www.cairographics.org/releases/cairo-1.14.10.tar.xz'
   source_sha256 '7e87878658f2c9951a14fc64114d4958c0e65ac47530b8ac3078b2ce41b66a09'
@@ -12,7 +12,7 @@ class Cairo < Package
   depends_on 'pixman'
 
   def self.build
-    system "./configure"
+    system "./configure --disable-xlib"
     system "make"
   end
 

--- a/packages/libsdl.rb
+++ b/packages/libsdl.rb
@@ -2,13 +2,13 @@ require 'package'
 
 class Libsdl < Package
   description 'Simple DirectMedia Layer is a cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware via OpenGL and Direct3D.'
-  homepage 'http://www.libsdl.org/'
+  homepage 'http://www.libsdl.org'
   version '1.2.15'
   source_url 'http://www.libsdl.org/release/SDL-1.2.15.tar.gz'
   source_sha256 'd6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00'
 
   def self.build
-    system "./configure"
+    system "./configure --disable-video-x11 --disable-video-wayland"
     system "make"
   end
 

--- a/packages/libsdl.rb
+++ b/packages/libsdl.rb
@@ -3,7 +3,7 @@ require 'package'
 class Libsdl < Package
   description 'Simple DirectMedia Layer is a cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware via OpenGL and Direct3D.'
   homepage 'http://www.libsdl.org'
-  version '1.2.15'
+  version '1.2.15-1'
   source_url 'http://www.libsdl.org/release/SDL-1.2.15.tar.gz'
   source_sha256 'd6d316a793e5e348155f0dd93b979798933fb98aa1edebcc108829d6474aad00'
 

--- a/packages/libsdl2.rb
+++ b/packages/libsdl2.rb
@@ -2,13 +2,13 @@ require 'package'
 
 class Libsdl2 < Package
   description 'Simple DirectMedia Layer is a cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware via OpenGL and Direct3D.'
-  homepage 'http://www.libsdl.org/'
+  homepage 'http://www.libsdl.org'
   version '2.0.5'
   source_url 'https://www.libsdl.org/release/SDL2-2.0.5.tar.gz'
   source_sha256 '442038cf55965969f2ff06d976031813de643af9c9edc9e331bd761c242e8785'
 
   def self.build
-    system "./configure"
+    system "./configure --disable-video-x11 --disable-video-wayland"
     system "make"
   end
 

--- a/packages/libsdl2.rb
+++ b/packages/libsdl2.rb
@@ -3,7 +3,7 @@ require 'package'
 class Libsdl2 < Package
   description 'Simple DirectMedia Layer is a cross-platform development library designed to provide low level access to audio, keyboard, mouse, joystick, and graphics hardware via OpenGL and Direct3D.'
   homepage 'http://www.libsdl.org'
-  version '2.0.5'
+  version '2.0.5-1'
   source_url 'https://www.libsdl.org/release/SDL2-2.0.5.tar.gz'
   source_sha256 '442038cf55965969f2ff06d976031813de643af9c9edc9e331bd761c242e8785'
 

